net-nds/openldap/openldap-2.4.44.ebuild

```diff
@@ -731,15 +757,15 @@ multilib_src_install() {
 
 		einfo "Installing contrib modules"
 		cd "${S}/contrib/slapd-modules" || die
-		for l in */*.la; do
+		for l in $(find -type f -name "*.la"); do
```

Comment:
have you checked whether these .la files are necessary? if the package installs `.pc` files, we don't need `.la` files and they should be nuked.